spec/polaris-management-service.yml

```diff
@@ -787,7 +799,7 @@ paths:
             schema:
               $ref: "#/components/schemas/RevokeGrantRequest"
       responses:
-        201:
+        204:
```

Comment:
This is technically a breaking change per our current "evolution" approach: https://polaris.apache.org/releases/1.0.1/evolution/#semantic-versioning

The change itself is meaningful, but if we make it, I suppose we'd have to do a major version bump (which is ok from my POV)..

Reply:
Removed as per ML - and will introduce it separately.